yt/frontends/ramses/field_handlers.py

```diff
@@ -263,6 +264,18 @@ def offset(self):
         self._level_count = level_count
         return self._offset
 
+    @classmethod
+    def load_fields_from_yt_config(cls) -> Tuple[List[str], bool]:
+        if cls.config_field and ytcfg.has_section(cls.config_field):
+            # Or this is given by the config
+            cfg = ytcfg.get(cls.config_field, "fields")
+            fields = []
+            for field in (_.strip() for _ in cfg.split("\n") if _.strip() != ""):
+                fields.append(field.strip())
```

Comment:
looks like a leftover from an intermediate step
```suggestion
            for field in (_.strip() for _ in cfg.split("\n") if _.strip() != ""):
                fields.append(field)
```